signing/signing.go

```diff
@@ -0,0 +1,51 @@
+package signing
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	SarifOutput string
+	JsonOutput  string
+}
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	// Get params to build GCS filepath.
+	ctx := context.Background()
+	bucketURL := "gs://ossf-scorecard-results"
+	host := mux.Vars(r)["host"]
+	orgName := mux.Vars(r)["orgName"]
+	repoName := mux.Vars(r)["repoName"]
+	resultsFile := filepath.Join(host, orgName, repoName, "results.json")
+
+	// Sanitize input and log query.
+	resultsFile = filepath.Clean(resultsFile)
+	escapedFilePath := strings.Replace(resultsFile, "\n", "", -1)
+	escapedFilePath = strings.Replace(escapedFilePath, "\r", "", -1)
```

Comment:
No need for these `strings.Replace` call. `filepath.Clean` should already take care of this.

Reply:
Removing these causes CodeQL scanning to fail: https://github.com/ossf/scorecard-webapp/pull/55/checks?check_run_id=5780026266

Reply:
Removing these causes CodeQL scanning to fail: https://github.com/ossf/scorecard-webapp/pull/55/checks?check_run_id=5780026266